src/js/components/Search.js

```diff
@@ -165,14 +165,21 @@ export default class Search extends Component {
       'bubbles': true,
       'cancelable': true
     });
-    this.refs.input.dispatchEvent(event);
-    this.props.onDOMChange(event.target.value, event);
+    var target = this.refs.input;
+    var value;
+    if (target) {
+      target.dispatchEvent(event);
+      value = event.target.value;
+    } else {
+      value = document.getElementById('search-drop-input').value;
+    }
+    this.props.onDOMChange(value);
```

Comment:
onDOMChange should be passing the event object, not just the value. This should match how SearchInput does it.